pybossa/syncer/project_syncer.py

```diff
@@ -55,7 +57,7 @@ def sync(self, project):
         else:
             raise NotEnabled
 
-        return res
+        return is_new_project, res
```

Comment:
Just wanted to make sure this was intended. Is the function returning two different result types?

If the project is new, it returns: `return self._create(payload, self.target_key)`
If the project exists, it returns: `is_new_project, res`

Reply:
That is fixed on the next PR that ill submit